deploy/install/01-prerequisite/06-storageclass.yaml

```diff
@@ -15,6 +15,7 @@ data:
     volumeBindingMode: Immediate
     parameters:
       fsType: "ext4"
+      mkfsParams: "-I 512 -b 2048 -O ^metadata_csum,^64bit"
```

Comment:
Should not change the `storageclass.yaml`. This is the default `longhorn` storageclass.
Add the example in the `longhorn/longhorn`'s `example/storageclass.yaml`

Reply:
I have fixed to remove the parameter `mkfs Params` from this yaml file.